command.lisp

```diff
@@ -338,9 +338,10 @@ then describes the symbol."
       ,@body)))
 
 (define-stumpwm-type :y-or-n (input prompt)
-  (let ((s (or (argument-pop input)
-               (read-one-line (current-screen) (concat prompt "(y/n): ")))))
-    (equal s "y")))
+  (let* ((valid '("y" "Y" t))
+         (s (or (argument-pop input)
+                (read-one-line (current-screen) (concat prompt "(y/n): "))))
+         (member s valid :test #'equal))))
```

Comment:
- valid is not correct, "n'"  is not invalid input, it is a negative response. 
-  we can use equalp to do case insensitive comparisons.

```suggestion
  (let* ((positive-responses '("y" t))
         (s (or (argument-pop input)
                (read-one-line (current-screen) (concat prompt "(y/n): "))))
         (member s positive-responses :test #'equalp))))
```